charts/k8ssandra/values.yaml

```diff
@@ -77,64 +77,12 @@ ingress:
       entrypoints:
         - cassandra
 
-    # Monitoring web interfaces for Promtheus and Grafana
-    monitoring:
-      grafana:
-        # Note this will **only** work if `ingress.traefik.enabled` is also `true`
-        enabled: true
-
-        # Name of the Traefik entrypoints where we want to source traffic.
-        entrypoints:
-          - web
-
-        # Hostname Traefik should use for matching requests.
-        host: grafana.k8ssandra.cluster.local
-
-      prometheus:
-        # Note this will **only** work if `ingress.traefik.enabled` is also `true`
-        enabled: true
-
-        # Name of the Traefik entrypoints where we want to source traffic.
-        entrypoints:
-          - web
-
-        # Hostname Traefik should use for matching requests.
-        host: prometheus.k8ssandra.cluster.local
-
 monitoring:
-  grafana:
-    security:
-      # TODO Add support for configuring credentials via a secret
-
-      # The admin username
-      adminUser: admin
-
-      # The admin password
-      adminPassword: secret
-
-    # Grafana server configuration settings. Corresponds to settings in /etc/grafana/grafana.ini.
-    config:
-      # Corresponds to properties in the server section of grafana.ini.
-      server:
-        # This is the full URL used to access Grafana from a web browser. This setting
-        # is also important if you have a reverse proxy in front of Grafana that
-        # exposes it through a subpath. In that case add the subpath to the end of this
-        # URL setting.
-        rootUrl: ""
-
-        # Set to to true to serve Grafana from subpath specified in rootUrl. If set to
-        # true, rootUrl should be set as well.
-        serveFromSubPath: false
+  grafana: 
+    provision_dashboards: true
 
   prometheus:
-    # The external URL the Prometheus instances will be available under.
-    externalUrl: ""
-
-    # The route prefix Prometheus registers HTTP handlers for. This is useful,
-    # if using ExternalURL and a proxy is rewriting HTTP routes of a request,
-    # and the actual ExternalURL is still true, but the server serves requests
-    # under a different route prefix. For example for use with `kubectl proxy`.
-    routePrefix: ""
+    provision_service_monitors: true
```

Comment:
This seems out of place to me. I think it would be more intuitive to be closer to the other Prometheus and/or promtheus-operator configs under `kube-prometheus-stack`.

Reply:
I placed it here as you may not have `kube-prometheus-stack` enabled at all (especially if we set a `condition` in `Chart.yaml`), but still want the service monitor to be provisioned (IE for centralized monitoring).

Reply:
Makes sense. Thanks for the explanation :)